src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -0,0 +1,104 @@
+import json
+import os
+import config_classes
+
+configurations_dir = 'asmcnc/apps/drywall_cutter_app/config/configurations'
+cutters_dir = 'asmcnc/apps/drywall_cutter_app/config/cutters'
+
+
+def fix_paths_for_tests():
+    global configurations_dir, cutters_dir
+    if os.getcwd().split('/')[-1] == 'easycut-smartbench':
+        configurations_dir = 'src/asmcnc/apps/drywall_cutter_app/config/configurations'
+        cutters_dir = 'src/asmcnc/apps/drywall_cutter_app/config/cutters'
+
+
+class DWTConfig(object):
+    active_config = None  # type: config_classes.Configuration
+    active_cutter = None  # type: config_classes.Cutter
+
+    def __init__(self):
+        fix_paths_for_tests()
```

Comment:
Can this be called from the test files? It seems like this risks causing havoc later on.

Reply:
Good idea, thanks

Reply:
Didn't work when I called it from the test instead :/

Reply:
Ok, then do a check to see if the paths exist before setting up the test case? Then don't need to comment anything out, and should be able to interchange between tests and actual use cases :)

Reply:
Done, it now changes the paths in the test file